### ISSUE TITLE
ImportError: cannot import name 'factorial' with pip installed packages

### ISSUE THREAD

**Issue by MarvinT** [statsmodels/statsmodels#5747]
#### Describe the bug

```pip install statsmodels scipy``` now installs incompatible versions. #5619 but now with the default installation failing. All the more reason to finish #5620.

#### Code Sample, a copy-pastable example if possible

```python
from statsmodels.formula.api import ols
```
or even
```python
import statsmodels.api as sm
```

#### Expected Output

pip installing the package shouldn't install incompatible types.

#### Output of ``import statsmodels.api as sm; sm.show_versions()``

<details>

>>> import statsmodels.api as sm; sm.show_versions()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/AD/mthielk/miniconda3/envs/tmp2/lib/python3.6/site-packages/statsmodels/api.py", line 16, in <module>
    from .discrete.discrete_model import (Poisson, Logit, Probit,
  File "/home/AD/mthielk/miniconda3/envs/tmp2/lib/python3.6/site-packages/statsmodels/discrete/discrete_model.py", line 45, in <module>
    from statsmodels.distributions import genpoisson_p
  File "/home/AD/mthielk/miniconda3/envs/tmp2/lib/python3.6/site-packages/statsmodels/distributions/__init__.py", line 2, in <module>
    from .edgeworth import ExpandedNormal
  File "/home/AD/mthielk/miniconda3/envs/tmp2/lib/python3.6/site-packages/statsmodels/distributions/edgeworth.py", line 7, in <module>
    from scipy.misc import factorial
ImportError: cannot import name 'factorial'

</details>

**Comment by bashtage** [comment]
You need to downgrade your scipy ot install statsmodels from master

**Comment by WarrenWeckesser** [comment]
I suspect there will be quite a few issues because of this.  See, for example, https://stackoverflow.com/questions/56206632/while-importing-auto-arima-from-pmdarima-error-cannot-import-name-factorial

**Comment by bashtage** [comment]
I agree. Hope to have one soonish (read 1 month).

**Comment by cpoptic** [comment]
What's the current fix to this?

**Comment by bashtage** [comment]
See #5759

**Comment by geethikabhavya** [comment]
Kindly let me know how to fix this - how do I downgrade scipy?

**Comment by MarvinT** [comment]
How did you install scipy? pip or conda?

**Comment by geethikabhavya** [comment]
Thank you. This worked for me : python3.6 -m pip install scipy==1.2 --upgrade
Credit: https://stackoverflow.com/questions/56283294/importerror-cannot-import-name-factorial

**Comment by SachitNayak** [comment]
Hi,

I'm still facing the issue, and I do not want to downgrade scipy... is the fix for the issue on the way for the next release?

**Comment by bashtage** [comment]
You should be able to do pip install statsmodels==0.10.0rc2 --pre  To get the release candidate that fixes this.

**Comment by SachitNayak** [comment]
@bashtage Thank you so much!

**Comment by clemenspeters** [comment]
1. `pip install statsmodels==0.10.0rc2 --pre`

 2. **Restarting the kernel** of the jupyter notebook

fixed it for me.<br>
You can check your versions with `pip list`

**Comment by ybenita1234** [comment]
> Thanks @bashtage
> 
> 1. `pip install statsmodels==0.10.0rc2 --pre`
> 2. **Restarting the kernel** of the jupyter notebook
> 
> fixed it for me.
> 
> You can check your versions with `pip list`

Thanks. It fixed the issue for me too.

**Comment by jorellano** [comment]
> Thanks @bashtage
> 
> 1. `pip install statsmodels==0.10.0rc2 --pre`
> 2. **Restarting the kernel** of the jupyter notebook
> 
> fixed it for me.
> 
> You can check your versions with `pip list`

Thank you. works now

**Comment by avitomar12** [comment]
> 1. pip install statsmodels==0.10.0rc2 --pre

Thanks @bashtage 
Worked for me.

**Comment by bashtage** [comment]
@avitomar12 You should use the latest release.  Just pip install statsmodels --upgrade.

**Comment by avitomar12** [comment]
@bashtage thanks
Yeah It worked. I used this in another machine.

**Comment by srivats0603** [comment]
thanks @bashtage. 
For other users: If you use databricks, then this dbutils command would work:
`dbutils.library.installPyPI('statsmodels','0.10.0rc2')`

**Comment by tknows20** [comment]
it seems that this bug is fixed in 0.11.1 version, https://www.statsmodels.org/stable/release/version0.11.1.html

**Comment by Manuelhrokr** [comment]
> it seems that this bug is fixed in 0.11.1 version, https://www.statsmodels.org/stable/release/version0.11.1.html

Thanks @tknows20. This happened to me with `statsmodels 0.9.0` and was able to fix it by reinstalling `statsmodels 0.12.2`. Probably the most recent releases should have this issue solved as well. 
If someone (like me) used `conda install -c conda-forge statsmodels` the first time you installed `statsmodels`, a clean new installation (I did it this way) can be make by: `conda remove -n yourenvhere statsmodels` first, then `conda list statsmodels` to be sure it was deleted from my environment, and finally `conda install statsmodels==0.12.2`. Furthermore, I am using `scipy 1.5.2`.
